exercises/grep/canonical-data.json

```diff
@@ -67,6 +67,33 @@
                 ]
             },
             {
+                "description": "One file, one match, print file names flag",
+                "pattern": "Forbidden",
+                "flags": ["-l"],
+                "files": ["paradise-lost.txt"],
+                "expected": [
+                    "paradise-lost.txt"
+                ]
+            },
+            {
+                "description": "One file, one match, match entire lines flag",
+                "pattern": "With loss of Eden, till one greater Man",
+                "flags": ["-x"],
+                "files": ["paradise-lost.txt"],
+                "expected": [
+                    "With loss of Eden, till one greater Man"
+                ]
+            },
+            {
+                "description": "One file, one match, multiple flags",
+                "pattern": "OF ATREUS, Agamemnon, KIng of MEN.",
+                "flags": ["-n", "-i", "-x"],
```

Comment:
I want to be careful about having this test for multiple flags before some of the flags have been tested individually. In general it seems that it would be good to allow students to get one flag working at a time, right? A more gradual progression to provide clearer steps to a full solution. But if this test is here, then the student would have to get `-n` and `-i` together (`-x` has been done before). It's also espoused in https://github.com/exercism/x-common/issues/230#issuecomment-237090138